its/ruling/src/test/resources/expected/python-S107.json

```diff
@@ -91,9 +91,6 @@
 510,
 519,
 ],
-'project:numpy-1.16.4/numpy/distutils/extension.py':[
-24,
```

Comment:
We can remove this FP as we should only raise an issue on the parent method and not the overridden one.